planner_cspace/src/planner_3d.cpp

```diff
@@ -826,6 +823,7 @@ class Planner3dNode
         }
       }
     }
+    map_update_retained_ = nullptr;
```

Comment:
This change is needed because `msg` is changed to reference.  `map_update_retained_` can be the same as `msg`, so `map_update_retained_` should be kept until `msg` is processed.